src/Components/ExternalResult/ResultList.tsx

```diff
@@ -239,7 +237,7 @@ export default function ResultList() {
           {value.name}
           <i
             className="fas fa-times ml-2 rounded-full cursor-pointer hover:bg-gray-500 px-1 py-0.5"
-            onClick={() =>
+            onClick={(_) =>
```

Comment:
```suggestion
            onClick={() =>
```

If event (`_`) is an optional one, so its better to be avoided when we don't use them